src/java_bytecode/expr2java.cpp

```diff
@@ -235,6 +237,23 @@ std::string expr2javat::convert_constant(
     dest+='L';
     return dest;
   }
+  else if (src.type() == java_float_type()) {
```

Comment:
could you please also run the `scripts/cpplint.py` script on this
For example, there should be no spaces around `=` or `==` or after `if`